myuw/test/dao/mailman.py

```diff
@@ -145,3 +161,42 @@ def test_get_section_label(self):
         self.assertEqual(get_section_label('ESS', '102', 'A',
                                            'spring', 2013),
                          '2013,spring,ESS,102/A')
+
+    def test__get_quarter_code(self):
+        self.assertEqual(_get_quarter_code('winter'), 1)
+        self.assertEqual(_get_quarter_code('spring'), 2)
+        self.assertEqual(_get_quarter_code("summer"), 3)
+        self.assertEqual(_get_quarter_code("autumn"), 4)
+
+    def test__get_single_line(self):
+        self.assertEqual(
+            _get_single_line(get_section_by_label(
+                    '2013,spring,PHYS,122/A')),
+            u'phys122a_sp13 2 2013 17983\n')
+
+    def test_get_message_body(self):
+        body, num1 = get_message_body('billsea',
+                                      ['2013,spring,PHYS,122/A',
+                                       '2013,spring,PHYS,122/AA',
+                                       '2013,spring,PHYS,122/AB',
+                                       '2013,spring,PHYS,122/AH'])
+        self.assertEqual(num1, 3)
+        self.assertEqual(body,
+                         (u'billsea\n' +
+                          u'phys122a_sp13 2 2013 17983\n' +
+                          u'phys122aa_sp13 2 2013 17984\n' +
+                          u'phys122ab_sp13 2 2013 17985\n'))
+
+    def test_request_mailman_lists(self):
+        self.assertEquals(len(mail.outbox), 0)
```

Comment:
Can you make it so this is wrapped by a settings override that forces the memory backend?  I want to be sure that running tests never sends emails